packages/storefront/components/checkout/paymentPage/subsections/paymentDetails.tsx

```diff
@@ -335,16 +354,7 @@ const PaymentDetails: NextComponentType = () => {
                       className="items-center text-sm font-semibold"
                     >
                       <label>
-                        <Field
-                          type="radio"
-                          name="shippingAddressPicked"
-                          value="sameShippingAddress"
-                          className="mx-4 mb-4 checked:accent-black"
-                          onClick={() => {
-                            handleSameAddress();
-                          }}
-                          required
-                        />
+                       w
```

Comment:
@SeefatHimel is it ok?

Reply:
Yes vai , last time you told us to comment this section (the address section in payment) .